docs/scheduler-http-api.md

```diff
@@ -128,6 +128,26 @@ TEARDOWN Response:
 HTTP/1.1 202 Accepted
 ```
 
+### SUPPRESS
+Sent by the scheduler when it wants to inform Mesos master to stop sending offers to the framework. When Mesos receives this request it will stop sending offers to framework but the tasks will continue running for `FrameworkInfo.failover_timeout`. Once the framework has work to do, it can send again a `REVIVE` request.
+
+```
+SUPPRESS Request (JSON):
+POST /api/v1/scheduler  HTTP/1.1
+
+Host: masterhost:5050
+Content-Type: application/json
+Mesos-Stream-Id: 130ae4e3-6b13-4ef4-baa9-9f2e85c3e9af
+
+{
+  "framework_id"        : {"value" : "12220-3440-12532-2345"},
+  "type"                        : "SUPPRESS"
+}
```

Comment:
The `Supress` call has a `roles` option. Please use an example call that sets a role to suppress offers for.